basic_pitch/note_creation.py

```diff
@@ -94,22 +104,33 @@ def model_output_to_notes(
         max_freq=max_freq,
         melodia_trick=melodia_trick,
     )
+
+    #apply the correction offset to align pitch (created corrected_notes instead of estimated notes)
+    corrected_notes = [
+        (note[0], note[1], int(note[2] - pitch_offset_correction), note[3], note[4])
```

Comment:
What is the implication of this cast to `int`? My understanding is that this truncates the decimal portion of the result, is this the desired behavior?